rpcs3/Emu/RSX/D3D12/D3D12GSRender.cpp

```diff
@@ -166,6 +167,7 @@ D3D12GSRender::D3D12GSRender()
 		// Adapter with specified name
 		if (adapter_name == desc.Description)
 		{
+			vram_size = desc.DedicatedVideoMemory != 0 ? desc.DedicatedVideoMemory : desc.DedicatedSystemMemory;
```

Comment:
I think its dedicated system memory for the graphics chip in this case. Think intel IGPs